configure.ac

```diff
@@ -2254,6 +2254,10 @@ else
 	CFLAGS="$curr_cflags"
 fi
 
+if test "$COB_USES_CLANG_ONLY" = yes; then
+	CFLAGS="$CFLAGS -Wno-unused-command-line-argument"
+fi
+
```

Comment:
this still should be set directly when COB_USES_CLANG_ONLY is set, otherwise we can get warnings on previous settings like hardening